pkg/cli/admin/release/extract_tools.go

```diff
@@ -445,10 +449,11 @@ func (o *ExtractOptions) extractCommand(command string) error {
 			NewArch: true,
 			Mapping: extract.Mapping{Image: "installer-artifacts", From: "usr/share/openshift/linux_amd64/openshift-install"},
 
-			Readme:               readmeInstallUnix,
-			InjectReleaseImage:   true,
-			InjectReleaseVersion: true,
-			ArchiveFormat:        "openshift-install-linux-amd64-%s.tar.gz",
+			Readme:                    readmeInstallUnix,
+			InjectReleaseImage:        true,
+			InjectReleaseVersion:      true,
+			InjectReleaseArchitecture: true,
```

Comment:
There are various usage patterns of this command, have you had a chance to check all of them;
* oc adm release info --tools
* oc adm release --command=installer --command=darwing/arm64
...

Reply:
yes, they will print a warning if the openshift-install doesn't have the right marker, but it will exit 0.